### PR TITLE
View Project

### LINKED ISSUE #2: View project

### PR DESCRIPTION
**Pull request by jackbravo**
Fixes #2

### CODE REVIEW

File: app/models/project.server.ts [hunk 1]
```diff
@@ -48,6 +50,51 @@ export class SearchProjectsError extends Error {
...
+      disciplines: true,
+      labels: true,
+      projectStatus: true,
+      owner: true,
+      projectMembers: {
+        include: {
+          profile: { select: { firstName: true, lastName: true, email: true } },
+          contributorPath: true,
+          practicedSkills: true,
+          //role: true,
```
Comment (LuisTejedaS) [inline app/models/project.server.ts:79]: I suggest delete commented code

File: app/routes/projects/$projectId.tsx [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+  console.error(error);
+
+  return <div>An unexpected error occurred: {error.message}</div>;
+}
+
+export function CatchBoundary() {
+  const caught = useCatch();
+
+  if (caught.status === 404) {
+    return <div>Note not found</div>;
```
Comment (LuisTejedaS) [inline app/routes/projects/$projectId.tsx:354]: I think here should say "Project not found"

File: app/models/project.server.ts [hunk 3]
```diff
@@ -1,8 +1,10 @@
-import type { Profiles } from "@prisma/client";
+import type { Profiles, Projects } from "@prisma/client";
 import { Prisma } from "@prisma/client"
 
 import { prisma as db } from "~/db.server";
 
+export type { Projects } from "@prisma/client";
```
Comment (LuisTejedaS) [inline app/models/project.server.ts:6]: I think this type is not used in the client side, why are we exporting it?

**Review by LuisTejedaS — COMMENTED** [review]
Added some comments

File: app/models/project.server.ts (hunk 1 above)
Comment (jackbravo) [reply app/models/project.server.ts:79]: I think this is instead an error that we haven't updated our schema and migrations from latest changes from the project-lab repo. I will work on this next.

File: app/models/project.server.ts (hunk 3 above)
Comment (jackbravo) [reply app/models/project.server.ts:6]: You are right, instead I later added the new type `ProjectComplete` extracted from the getProject function.

**Review by LuisTejedaS — APPROVED** [review]
LGTM

**Pull request merged**